model/jpa/pom.xml

```diff
@@ -84,7 +84,50 @@
                     <groupId>com.h2database</groupId>
                     <artifactId>h2</artifactId>
```

Comment:
ok, if H2 is not needed, let's remove the dependency

On Fri, Jan 23, 2015 at 12:51 PM, Erik Jan de Wit <notifications@github.com>
wrote:

> In model/jpa/pom.xml
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/473#discussion_r23445422>
> :
>
> > @@ -84,7 +84,50 @@
> >                      <groupId>com.h2database</groupId>
> >                      <artifactId>h2</artifactId>
>
> we could use two, but that would make things more complex
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/473/files#r23445422>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf